daemon/src/command.rs

```diff
@@ -0,0 +1,47 @@
+use crate::cfd_actors::load_cfd;
+use crate::model::cfd::Cfd;
+use crate::model::cfd::Event;
+use crate::process_manager;
+use crate::OrderId;
+use anyhow::Context;
+use anyhow::Result;
+use xtra::Address;
+
+pub struct Executor {
```

Comment:
it would actually be lovely if you inserted a comment here - even that commit description! (good for both IDEs and generated docs)

Reply:
BTW nice abstraction!